schemas/2015-08-01/Microsoft.Cache.json

```diff
@@ -0,0 +1,160 @@
+            "description": "The resource ID of an Azure Classic Virtual Network"
+        },
+        "ipV4Address": { 
+            "type": "string",
+            "pattern": "^\\d+\\.\\d+\\.\\d+\\.\\d+$",
+            "description": "An IP address in a Virtual Network"
+        },
+        "Redis": {
+            "type": "object",
+            "properties": {
+                "type": {
+                    "enum": [
+                        "Microsoft.Cache/Redis"
+                    ]
+                },
+                "apiVersion": {
+                    "enum": [
+                        "2015-08-01"
+                    ]
+                },
+                "properties": {
+                    "type": "object",
+                    "properties": {
+                        "sku": {
+                            "type": "object",
+                            "properties": {
+                                "name": {
+                                    "oneOf": [
+                                        {
+                                            "enum": [
+                                                "Basic",
+                                                "Standard",
+                                                "Premium"
+                                            ]
+                                        },
+                                        {
+                                            "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+                                        }
+                                    ],
+                                    "description": "Microsoft.Cache/Redis/sku: determines features and pricing of the cache",
+                                    "default": "Standard"
+                                },
+                                "family": {
+                                    "oneOf": [
+                                        {
+                                            "enum": [
+                                                "C",
+                                                "P"
+                                            ]
+                                        },
+                                        {
+                                            "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+                                        }
+                                    ],
+                                    "description": "Microsoft.Cache/Redis/sku: along with capacity, determines capacity of the cache",
+                                    "default": "C"
+                                },
+                                "capacity": {
+                                    "oneOf": [
+                                        {
+                                            "type": "integer"
```

Comment:
I don't think it's worth changing it. I would expect tools will actually be better at supporting integer than positiveInteger, and chance of user error here seems minimal.